vendor/github.com/docker/docker/pkg/system/stat_linux.go

```diff
@@ -8,7 +8,8 @@ func fromStatT(s *syscall.Stat_t) (*StatT, error) {
 		mode: s.Mode,
 		uid:  s.Uid,
 		gid:  s.Gid,
-		rdev: s.Rdev,
+		// the type is 32bit on mips
+		rdev: uint64(s.Rdev), // nolint: unconvert
```

Comment:
Looks like this change was included in https://github.com/docker/cli/commit/38c02a9280ed8159e323a6df03bc49f9c6cb086c (https://github.com/docker/cli/pull/2110)